.travis.yml

```diff
@@ -5,16 +5,24 @@ os:
   - osx
 julia:
   - 0.6
+  - 0.7
+  - 1.0
   - nightly
 matrix:
   allow_failures:
     - julia: nightly
   fast_finish: true
 notifications:
   email: false
-script:
-  - if [[ -a .git/shallow ]]; then git fetch --unshallow; fi
-  - julia -e 'Pkg.clone(pwd()); Pkg.build("KeyedFrames"); Pkg.test("KeyedFrames"; coverage=true)'
+# uncomment the following lines to override the default test script
+# script:
+#   - if [[ -a .git/shallow ]]; then git fetch --unshallow; fi
+#   - julia -e 'Pkg.clone(pwd()); Pkg.build("KeyedFrames"); Pkg.test("KeyedFrames"; coverage=true)'
 after_success:
-  - julia -e 'cd(Pkg.dir("KeyedFrames")); Pkg.add("Coverage"); using Coverage; Codecov.submit(process_folder())'
-  - julia -e 'Pkg.add("Documenter"); cd(Pkg.dir("KeyedFrames")); include(joinpath("docs", "make.jl"))'
+  # push coverage results to Codecov
```

Comment:
Simon Byrne added a nifty thing so that coverage submission can be simplified to
```yaml
codecov: true
```
which submits coverage in a cross-version manner (as you've done manually here) after success. Something to try out the next time we update this stuff in a public package.